wiki/Building-on-Linux.md

```diff
@@ -40,26 +45,38 @@ sudo dnf groupinstall "Development Tools"
 sudo dnf install 'dnf-command(config-manager)'
 sudo dnf config-manager --set-enabled PowerTools
 sudo dnf install cmake3 avahi-compat-libdns_sd-devel libX11-devel libXtst-devel \
-                qt5-qtbase-devel libcurl-devel desktop-file-utils openssl-devel
+                qt6-qtbase-devel libcurl-devel desktop-file-utils openssl-devel
 ```
 
-### openSUSE
+### openSUSE## Building the Project
 
 ```shell
 sudo zypper install --type pattern devel_basis
 sudo zypper install libdrm-devel libglvnd-devel libICE-devel \
-        libQt5Core-devel libQt5Gui-devel libQt5Network-devel \
-        libqt5-qtbase-common-devel libQt5Widgets-devel libSM-devel \
+        libQt6Core-devel libQt6Gui-devel libQt6Network-devel \
```

Comment:
Are your sure these are the right packages? I don't think they exist

Reply:
pkgconfig(Qt6Core)
pkgconfig(Qt6Gui)
pkgconfig(Qt6Network)
pkgconfig(Qt6Platform)
pkgconfig(Qt6Widgets)

I think it also needs pkgconfig(Qt6Linguist)

Reply:
How about replacing most by pkgconfig?